plotter/image_test.go

```diff
@@ -0,0 +1,86 @@
+// Copyright ©2016 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package plotter
+
+import (
+	"fmt"
+	"image/png"
+	"log"
+	"os"
+	"testing"
+
+	"github.com/gonum/plot"
+	"github.com/gonum/plot/vg"
+	"github.com/gonum/plot/vg/draw"
+	"github.com/gonum/plot/vg/vgtex"
+)
+
+const runImageLaTeX = false
+
+// An example of embedding an image in a plot.
+func ExampleImage() {
+	p, err := plot.New()
+	if err != nil {
+		log.Fatalf("error: %v\n", err)
+	}
+	p.Title.Text = "The Horsehead Nebula"
+
+	// load an image
+	f, err := os.Open("testdata/horsehead.png")
+	if err != nil {
+		log.Fatalf("error opening image file: %v\n", err)
+	}
+	defer f.Close()
+
+	img, err := png.Decode(f)
+	if err != nil {
+		log.Fatalf("error decoding image file: %v\n", err)
+	}
+
+	p.Add(NewImage(img, 100, 100, 200, 200))
+	p.Add(NewGlyphBoxes())
```

Comment:
No need for this. Drawing the glyphboxes is just for debugging them. If someone wants to display an image, you don't want them copying this line.